Dockerfile

```diff
@@ -1,13 +1,14 @@
-FROM golang:1.22.1-alpine as build
+FROM --platform=$BUILDPLATFORM golang:1.22.1-alpine as build
 
-COPY *.go /go/src/github.com/nanoandrew4/ngcplogs/
-COPY go.mod /go/src/github.com/nanoandrew4/ngcplogs/
-COPY go.sum /go/src/github.com/nanoandrew4/ngcplogs/
-
-RUN cd /go/src/github.com/nanoandrew4/ngcplogs && go get && go build --ldflags '-extldflags "-static"' -o /usr/bin/ngcplogs
+WORKDIR /goapp
+COPY go.mod go.sum ./
+RUN --mount=type=cache,target=/root/.cache/go-build --mount=type=cache,target=/go/pkg go mod download
+COPY *.go ./
+ARG TARGETOS TARGETARCH
```

Comment:
Each arg should exist on a separate line AFAIK, and the docker reference seems to suggest the same